rpkg_src/thirdparty/zhmtools/Embind.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#if __EMSCRIPTEN__
```

Comment:
This file shouldn't be needed, as RPKG can't work in a browser environment anyway.

Reply:
You could argue the same for HMA. But it's being kept as it's directly from ResourceLib's include folder.